.github/workflows/tests_lgpumpi_python.yml

```
@@ -105,13 +105,14 @@ jobs:
           echo "pip=$pip_path" >> $GITHUB_OUTPUT
 
       - name: Install required packages
+        env:
+          PL_BACKEND: lightning_qubit
         run: |
           source /etc/profile.d/modules.sh && module use /opt/modules/ && module load ${{ matrix.mpilib }}/cuda-${{ matrix.cuda_version_maj }}.${{ matrix.cuda_version_min }}
           python -m pip install -r requirements-dev.txt
 
           python -m pip install custatevec-cu${{ matrix.cuda_version_maj }} mpi4py openfermionpyscf
-          PL_BACKEND=lightning_qubit python scripts/configure_pyproject_toml.py
-          python -m pip install . -vv
+          python scripts/configure_pyproject_toml.py || python -m pip install . -vv
```

Don't we need `|| true`?
```suggestion
          python scripts/configure_pyproject_toml.py || true
          python -m pip install . -vv
```

---

yes, I lost this one. thanks!